homeassistant/components/cpuspeed/sensor.py

```diff
@@ -32,27 +44,29 @@ class CpuSpeedSensor(SensorEntity):
     _attr_native_unit_of_measurement = FREQUENCY_GIGAHERTZ
     _attr_icon = "mdi:pulse"
 
-    def __init__(self, name):
+    def __init__(self, name: str) -> None:
         """Initialize the CPU sensor."""
         self._attr_name = name
-        self.info = None
+        self.info: dict[str, Any] | None = None
```

Comment:
Since `self.info` is only used for the `extra_state_attributes`, could you set `self._attr_state_attributes` directly inside the `update` method?

Reply:
(I missed it in #62896)

Reply:
It will be refactored out in follow up PRs; so that is a waste of effort (hence I didn't do it)